angular2-jwt.ts

```diff
@@ -304,56 +304,34 @@ export function tokenNotExpired(tokenName = AuthConfigConsts.DEFAULT_TOKEN_NAME,
   return token != null && !jwtHelper.isTokenExpired(token);
 }
 
-export const AUTH_PROVIDERS: Provider[] = [
```

Comment:
@escardin A lot of things are documented in here https://github.com/rangle/angular-2-aot-sandbox